flyway/sql/V1__create_tables.sql

```diff
@@ -59,3 +59,7 @@ CREATE TABLE IF NOT EXISTS `ethnicity`
   `identifier` VARCHAR(100) NOT NULL
 ) CHARACTER SET utf8 COLLATE utf8_general_ci;
 
+-- Alter answer table to create composite unique for (visitor id ,question id)
+
+ALTER TABLE `mpop_database`.`answer` DROP INDEX `visitor_question_id`,
+ADD UNIQUE `visitor_question_id` (`visitor_id`, `question_id`) USING BTREE;
```

Comment:
I believe this is a duplicate from the other PR. Each PR should have its own branch and not contain the other branche's changes.

Reply:
Ohh I guess it it added commit from the previous branch